### DISCUSSION TITLE
Dagster not identifying packages installed with pipenv

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17604] [Q&A]
I use pipenv for my virtual environments, so I ran in a new project:
pipenv install
dagster dev -f ./main.py

But dagster failed since it wasn't able to find the packages I installed using pipenv within the run.
Is there any way for me to make it identify the packages installed in this virtual environment?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1698858778446409?thread_ts=1698858778.446409&cid=C01U954MEER).*

**Comment by shalabhc** [comment]
Have you activated the virtual environment so the same pipenv environment is used to launch the `dagster` command?

You can also try `pipenv run dagster dev -f ./main.py` to ensure that `dagster` is run from the same environment.
